plugins/modules/win_package.py

```diff
@@ -39,6 +39,24 @@
       or uninstalling a package.
     - This is only used for the C(msi), C(msp), and C(registry) providers.
     type: path
+  checksum:
+    description:
+      - If a I(checksum) is passed to this parameter, the digest of the
+        package will be calculated before executing it to verify that the
+        transfer completed successfully.
```

Comment:
This is slightly confusing, maybe change transfer completed successfully to something like

> verify that the path or downloaded file has the expected contents